File: LeetCode/Easy/0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        x=str(x)
        n=len(x)
        x=int(x)
        real=x
        temp=0
        for i in range(n):
            a=x%10
            temp= temp*10 + a
            x=x//10
        if temp==real:
            return True
        else:
            return False
        

File: LeetCode/Easy/0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let rev=0
    let temp
    let temp1=x
    let n = String(x).length
    for(let i=0;i<n;i++){
        temp=x%10
        rev=rev*10 + temp
        x=Math.floor(x/10)
    }
    if (temp1==rev){
        return true 
    }
    else{
        return false
    }
    
};